persistence/upgrades/ravendb-4to5.md

```diff
@@ -48,3 +48,9 @@ When using this setting, RavenDB Persistence will log a warning at each endpoint
 > RavenDB Persistence is using legacy versioned subscription storage. This capability will be removed in NServiceBus.RavenDB 6.0.0. Subscription documents need to be converted to the new unversioned format, after which `persistence.DisableSubscriptionVersioning()` should be used.
 
 For more details, see [Subscription versioning](/persistence/ravendb/subscription-versioning.md?version=raven_5).
+
+## Multi-tenant support
+
+Starting with version 5 it's possible to select the database used to store NServiceBus related data, such as Saga data and Outbox records, based on information stored in the incoming messages headers:
```

Comment:
Snippet is good, but I'm not sure if the upgrade guide is the right place. We added it [in 4.2](https://github.com/Particular/NServiceBus.RavenDB/blob/support-4.2/src/NServiceBus.RavenDB/RavenDbSettingsExtensions.cs#L118-L122) as well. Well, the [API exists in 4.1](https://github.com/Particular/NServiceBus.RavenDB/blob/support-4.1/src/NServiceBus.RavenDB/RavenDbSettingsExtensions.cs#L99-L103) but it set a static member variable so that was silly and didn't work, especially (I seem to recall) if you wanted to use Outbox.

So should we leave it out of the upgrade guide and document for 4.2 and 5.x instead?

Reply:
Good idea, I’ll have a look tomorrow. Or if you have time feel free to change it during your day.